client/src/App.jsx

```diff
@@ -22,6 +25,39 @@ function App() {
     getAllFromApi()
   }, [])
 
+  const [messages, setMessages] = useState([])
```

Comment:
I think it's a convention to write all useStates on top of the function, but of cause this is a refactoring task for later.